### PR TITLE
Plugin on click

### PR DESCRIPTION
**Pull request by KonradLinkowski**
I created the plugin handling click events as mentioned in #8 .
If you have any suggestions I can change it.

### CODE REVIEW

**Comment by bryanbraun** [comment]
Taking a look now. Thanks for your patience!

File: docs/demos/on-click/demo.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+let dimensions = '44x15';
+let cbl;
+let lastPos = null;
+
+function init(existingCbl) {
+  cbl = !!existingCbl ? existingCbl : new Checkboxland({ dimensions });
+
+  const data = cbl.getData();
+  cbl.setData(data.map(e => e.map(_ => 0)));
```
Comment (bryanbraun) [inline docs/demos/on-click/demo.js:11]: This works fine (no need to update this), but just to let you know, you could also use [`cbl.clearData()`](https://www.bryanbraun.com/checkboxland/#cleardata) for this. 👍

File: docs/demos/on-click/demo.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+function cleanUp() {
+  cbl.onClick.cleanUp();
+}
+
+
+export {
+  init,
+  cleanUp,
+  dimensions,
+}
```
Comment (bryanbraun) [inline docs/demos/on-click/demo.js:55]: I'm so happy you included a demo. It works great on both the docs home page and the standalone page. You even included a screenshot, and cleans up the events after it's done. Stellar work. 💯

File: src/plugins/onClick.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      if (checkbox === target) {
+        return { x, y };
+      }
+    }
+  }
+  return null;
+}
+
+function cleanUp() {
+  displayEl.removeEventListener('click', handleFun);
```
Comment (bryanbraun) [inline src/plugins/onClick.js:45]: This works great, and is exactly what `cleanUp()` was designed for. 👍

Comment (bryanbraun) [reply src/plugins/onClick.js:45]: Two more things I wanted to mention:
- Kudos for using event delegation, to make sure that we only needed one event for the whole grid. 🏆 
- I didn't event know about the [`EventListener` object interface](https://developer.mozilla.org/en-US/docs/Web/API/EventListener), but you added support for it and a nice error message. I learned something new today. 😄

Comment (KonradLinkowski) [reply src/plugins/onClick.js:45]: I didn't know about thet `EventListener` object either for a long time. I learned about it few months ago when I was creating an `addEventListener` implementation in TypeScript.
I'm glad that you've learned something from my code :)

**Pull request merged**